### DISCUSSION TITLE
Votecounter for daylie votes over the whole server. daylie - weekly - monthly

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#3857] [Feature Requests]
We allready have a list to see all votes one single user was voteing.. for one day
but we also nice to have one for the player for **weekly and monthly** statistics?
Well these are for singleusers. 

But what we also would need is one for  the whole server?
**daylie, weekly, monthly**
its more interesting how much players have voted for the whole server a day, week, month

but the daylie user one is also nice for some nice interactions.

**Comment by thboehi** [comment]
Good idea :)
